_data/locales/hu.yml

```diff
@@ -0,0 +1,31 @@
+hu:
+  locale_name: Magyar
+  nav:
+    about: Rólunk
+    contribute: Közreműködés
+  index:
+    lead: A nyílt forráskódú szoftvereket ugyanolyan emberek készítik mint te. Tanuld meg, hogy hogyan indíthatod el és hogyan növekedhet a projekted!
+    opensourcefriday: Péntek van! Szánj pár órát a kedvenc szoftveredre amit használsz!
+  article:
+    table_of_contents: Tartalomjegyzék
+    back_to_all_guides: Vissza az Útmutatókra
+    related_guides: Kapcsolódó Útmutató
+  footer:
+    contribute:
+      heading: Közreműködés
+      description: Van javaslatod? Ez a tartalom is nyílt forráskód, segíts benne!
+      button: Közreműködés
+    subscribe:
+      heading: Maradjunk kapcsolatban
+      description: Legyél Te az első, aki hall a GitHub's legfissebb nyílt forráskódú híreiről és anyagairól
+      label: Email cím
+      button: Feliratkozás
+    byline:
+      # [code], [love], and [github] will be replaced by octicons
+      format: "[code] with [love] by [github] and [friends]"
```

Comment:
@patoi Worth translating these (and below)?

Reply:
If you agree: a follow-up PR would be lovely 💖

Reply:
Ok, thank you!